docs/advanced.md

```diff
@@ -314,6 +314,23 @@ Although it kind of defeats the purpose of using them in the first place (not to
 ENV["DELIVER_USER"] = "felix@krausefx.com"
 ```
 
+# Lane Properties
+
+It can be useful to dynamically access properties of the current lane. These are available in `lane_context`
+
+```ruby
+lane_context[SharedValues::PLATFORM_NAME]        # Platform name, e.g. `ios`, `android` or empty (for root level lanes)
+lane_context[SharedValues::LANE_NAME]            # The name of the current lane (stays the same when switching lanes, includes platform name)
```

Comment:
I'd personally change it to `, applies to platform name too` or even better, move it to the previous line so even if it'd be duplicated, it would be clearer.

Reply:
Hm, "includes platform name" is not meant as a qualifier to "stays the same...". The lane name actually includes the platform name: `ios foo` for lane `foo` for platform `ios`.

Reply:
(not actually outdated @Github...)

Reply:
Ohhh, I thought that the platform stays the same. Maybe

> The name of the current lane preceded by the platform name.

? How does it sound to you?

Reply:
Good.